en/flying/terrain_following_holding.md

```diff
@@ -0,0 +1,40 @@
+# Terrain Following/Holding
+
+PX4 supports *Terrain Following* and *Terrain Hold* on **multicopters** that have a [distance sensor](../sensor/rangefinders.md).
+
+The features are available in manual altitude-control modes: [Position](../flight_modes/position_mc.md), [Altitude](../flight_modes/altitude_mc.md).
+They are not available in acrobatic modes or automatic modes (e.g. Return, Land, Mission, etc.).
```

Comment:
- `acrobatic modes` I would take that to be obvious, no?
- `automatic modes`: I would document (or link) EKF2_HGT_MODE/EKF2_RNG_AID here as well, to make clear we support terrain following in missions too (after we clarified how/if that all works).